components/AttemptOverview.vue

```diff
@@ -98,6 +105,39 @@ export default class AttemptOverview extends Vue {
     return this.attemptSplitTimesaves.reduce((acc: number, n: number | null) => acc + (n || 0), 0);
   }
 
+  segmentNameFormat(name: string) {
+    if (name.startsWith('-'))
+      return name.substring(1);
+    
+    if (name.startsWith('{')){
+      const cutIndex = name.indexOf('}') ;
+      const splitName = name.substring(1, cutIndex);
+      const subsplitName = name.substring(cutIndex + 2);
+
+      return `${splitName} | ${subsplitName}`;
+    }
+
+    return name;
+  }
+
+  get segmentLabels() {
+    const segmentData = this.mergeSubsplits ? this.detailedSegments : this.run.Segments.Segment;
+    return segmentData.map(segment => {
+        const time = selectTime((segment.SegmentHistory?.Time || []).find(t => t['@_id'] == this.attempt['@_id']));
+        const segmentName = this.segmentNameFormat(segment.Name)
+        return time ? `${segmentName} (${formatTime(time)})` : segmentName;
+    });
+  }
+
+  get timesaveLabels() {
+    const segmentData = this.mergeSubsplits ? this.detailedSegments : this.run.Segments.Segment;
+    return segmentData.map((segment, i) => {
+      const ast = this.attemptSplitTimesaves[i];
```

Comment:
The `ast` value you're getting here is incorrect if the `detailedSegments` are used, because the `detailedSegments` array is shorter than the segments array from which `attemptSpltTimesaves` are computed, and therefore the index don't match, this results in timesave not changing when flipping the switch (in particular look how VS has the KP timesave even though it should be more than 3s, and all the following timesaves are offset)

![image](https://user-images.githubusercontent.com/12281309/169376184-017dc0fd-f6b6-4013-83eb-5d23479567cb.png)
-> 
![image](https://user-images.githubusercontent.com/12281309/169376313-4b8ef2ec-fb70-4d44-a124-547b23f6a8cd.png)

This is because `attemptSpltTimesaves` is computed from `run.Segments.Segment`, it should be either that or from `detailedSegments` depending on the switch value

Reply:
Also it is very suspicious for the total timesave to stay the same, because it's very unlikely to get a gold on all the subsplits of the same group on the same run. Usually more subsplits mean lowering the sum of best (because each segment is smaller), so when merging splits I think the total possible timesave should decrease